packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -92,10 +88,25 @@ export class MarketOperationUtils {
                 this._liquidityProviderRegistry,
             ),
         );
+        const rfqtPromise =
+            opts !== undefined && _opts.rfqt !== undefined && _opts.rfqt.quoteRequestor !== undefined
```

Comment:
Since we merge defaults into `_opts`, and never access it further down, do we need the `opts !== undefined` check?

Reply:
This was a bug, that I noticed too, and fixed here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/d6d4d29257c4cb0810c280f8edc33939f0e6092e#diff-bdb1465a97651026c09ef607e5de9e6fR97